h/api/subscribers.py

```diff
@@ -10,7 +10,7 @@ def index_annotation_event(event):
     if not event.request.feature('postgres'):
         return
 
-    if event.action == 'create':
+    if event.action in ['create', 'update']:
```

Comment:
Depending on which PR gets merged first (see #3242), this has to move into the new `h.indexer`